lms/templates/courseware/accordion.html

```diff
@@ -6,73 +6,44 @@
 %>
 
 <%def name="make_chapter(chapter)">
-  <div class="chapter">
-      <%
-          if chapter.get('active'):
-              aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
-              active_class = ' class="active"'
-          else:
-              aria_label = chapter['display_name']
-              active_class = ''
-      %>
-      <h3 ${active_class} aria-label="${aria_label}">
-        <a href="#">
-          ${chapter['display_name']}
-        </a>
-      </h3>
-
-    <ul>
-      % for section in chapter['sections']:
-          <li class="${'active' if 'active' in section and section['active'] else ''} ${'graded'  if 'graded' in section and section['graded'] else ''}">
+<%
+if chapter.get('active'):
+    aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
+    active_class = 'active"'
+else:
+    aria_label = chapter['display_name']
+    active_class = ''
+%>
+<button class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-pressed="false">
+    <h3 class="group-heading ${active_class}" aria-label="${aria_label}">
+        <span class="icon fa fa-caret-right" aria-hidden="true"></span>
+        ${chapter['display_name']}
+    </h3>
+</button>
+<div class="chapter-content-container" tabindex="-1" aria-expanded="false">
```

Comment:
@clrux is there any reason why we can't consolidate this div with the one below?  I worry that this one has a tabindex-1 and the next has a role of region, meaning they may both result in being reported by SR.  I think we can consolidate all the properties. Bring the role, aria-labelledby and aria-hidden from below up here and get rid of the div below.